pkgs/os-specific/linux/libselinux/default.nix

```diff
@@ -35,7 +35,6 @@ stdenv.mkDerivation rec {
     "MAN3DIR=$(man)/share/man/man3"
     "MAN5DIR=$(man)/share/man/man5"
     "MAN8DIR=$(man)/share/man/man8"
-    "PYSITEDIR=$(py)/${python.sitePackages}"
```

Comment:
You can use `PYTHONLIBDIR`: https://github.com/SELinuxProject/selinux/blob/dc3d9c6d7173354cc4caf225ce75c41239cf47bd/libselinux/src/Makefile#L17

Reply:
Oh, yeah, thanks! Will fix this in the next push.